modules/javafx.graphics/src/main/java/javafx/css/Stylesheet.java

```diff
@@ -293,10 +294,10 @@ public static Stylesheet loadBinary(URL url) throws IOException {
     }
 
     /**
-     * Convert the .css file referenced by urlIn to binary format and write to urlOut.
-     * @param source is the JavaFX .css file to convert
-     * @param destination is the file to which the binary conversion is written
-     * @throws IOException the exception
+     * Convert the .css file referenced by source to binary format and write to destination.
+     * @param source the JavaFX .css file to convert
+     * @param destination the file to which the binary version is written
```

Comment:
"binary formatted data" is probably better than "binary version" because the latter can be mistaken for a version number